metadata-ingestion/src/datahub/ingestion/source/sql/presto_on_hive.py

```diff
@@ -4,6 +4,8 @@
 from collections import namedtuple
 from itertools import groupby
 from typing import Any, Dict, Iterable, List, Optional, Tuple, Union
+from enum import Enum
+from wsgiref import validate
```

Comment:
This looks like an unused import. Please follow the instructions [here](https://datahubproject.io/docs/metadata-ingestion/developing/#sanity-check-code-before-committing) for a quick sanity check.

Reply:
Thanks for sharing